packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -457,7 +486,7 @@ export default class ListControl extends React.Component {
         <StyledListItemTopBar
           collapsed={collapsed}
           onCollapseToggle={partial(this.handleItemCollapseToggle, index)}
-          onRemove={partial(this.handleRemove, index, key)}
+          onRemove={allowRemove ? partial(this.handleRemove, index, key) : null}
```

Comment:
Would it better to have an explicit error when saving than removing the buttons?
I find it quite confusing for those buttons to suddenly disappear without any clear explanation why.

Reply:
see new commit